contrib/ceph/micro-osd.sh

```diff
@@ -0,0 +1,90 @@
+#
+#    Copyright (C) 2013,2014 Loic Dachary <loic@dachary.org>
```

Comment:
@gierschv I've been following along on this PR as I'm quite interested in getting rados support into this next gen registry. 

On this issue of testing: the cephfs-hadoop (https://github.com/ceph/cephfs-hadoop) project has some test scripts and a Vagrant script they use in testing which might have a more compatible license and satisfy your test needs. 

If you haven't already checked that project out, I are commend you should.